<launch>

  <!-- Low level steering controller for 6WD car-->
  <node pkg="low_level_steering_controller" name="steer_controller" type="six_wheel_low_level_steering_controller" required="true" output="screen">
    <param name="steer_topic_name" type="string"   value="/motor_controller/motor_commands"/>
  </node>

</launch>

